.changes/next-release/feature-AWSSDKforJavav2-0e09833.json

```diff
@@ -0,0 +1,6 @@
+{
+    "category": "AWS SDK for Java v2", 
+    "contributor": "", 
+    "type": "feature", 
+    "description": "Add new AsyncResponseTransformer: toPublisher(). This transformer allows users to directly consume a streaming-response payload (i.e., S3 GetObject) without having to buffer to memory or disk. This also allows users of Reactor/RxJava to more easily consume a streaming response (e.g., via Flux#from(Publisher))."
```

Comment:
Yes, it is technically possible, but many users don't wish to implement their own `AsyncResponseTransformer`, and there are certain assumptions the SDK makes about the transformer's `CompletableFuture`-completion that don't play nicely together (e.g., metric publishing). But will clarify the wording to indicate that we are providing first-class support for it rather than making it technically possible.